internal/docstore/firedocstore/codec.go

```diff
@@ -14,6 +14,21 @@
 
 package firedocstore
 
+// Encoding and decoding between supported docstore types and Firestore protos.
```

Comment:
Nit: where does this show up in godoc? Should it be in the package docstring instead?

Reply:
Temporarily made a package docstring for this purpose. Will move it to `fs.go` later.

Reply:
PTAL.